invenio/ext/sqlalchemy/__init__.py

```diff
@@ -194,6 +194,14 @@ def apply_driver_hacks(self, app, info, options):
                 'charset': 'utf8mb4',
             })
             event.listen(Pool, 'checkin', autocommit_on_checkin)
+        elif info.drivername == 'postgresql':
+            from psycopg2.extensions import register_adapter, AsIs
+            from werkzeug.local import LocalProxy
+
+            def adapt_proxy(proxy):
+                return AsIs("'%s'" % proxy._get_current_object())
```

Comment:
The way I see it, they are correct: what `adapt_proxy` is doing is unwrapping the string from the `LocalProxy` and wrapping it in a `AsIs` object, which has the required method `getquoted` that PostgreSQL expects.